spec/models/reviewable_flagged_post_spec.rb

```diff
@@ -99,6 +99,20 @@ def pending_count
           expect(reviewable.actions_for(guardian).has?(:delete_user_block)).to be true
         end
       end
+
+      context "for ignore_and_do_nothing" do
+        it "does not return `ignore_and_do_nothing` when post is hidden" do
+          post.update(hidden: true)
```

Comment:
I think we just also need to test that `expect(reviewable.actions_for(system_guardian).has?(:ignore_and_do_nothing)).to eq(true)` is true when the post is hidden and the user is system

Reply:
Oops. that was totally my intention 🤪